tests/unit/test_plugins/test_pydantic/test_plugin_serialization.py

```diff
@@ -198,9 +217,26 @@ def model(pydantic_version: PydanticVersion) -> ModelV1 | ModelV2:
         ("conint", 1),
         ("url", "some://example.org/"),
         ("http_url", "http://example.org/"),
+        ("paste_date", YESTERDAY_DATE.isoformat()),
```

Comment:
You'll need to make these values constant, otherwise xdist will trip up because they're different on the different workers